catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/ResultDAGConverter.java

```diff
@@ -819,8 +819,7 @@ public static DAG convertResult(Result result, ORB orb, POA poa, List<String> re
                 addStringAttribute(graph, relatedFileNode, NsiliConstants.URL, thumbnailURL, orb);
                 addedAttributes.add(buildAttr(attribute, NsiliConstants.URL));
             } catch (URISyntaxException e) {
-                LOGGER.warn("Unable to construct URI: {}", e);
-                LOGGER.debug("", e);
+                LOGGER.debug("Unable to construct URI: {}", e);
```

Comment:
This will never give us the stack trace... remove the {}... or better yet, add the string it failed to make a URI out of